editor/js/Editor.js

```diff
@@ -8,6 +8,10 @@ var Editor = function () {
 	this.DEFAULT_CAMERA.name = 'Camera';
 	this.DEFAULT_CAMERA.position.set( 0, 5, 10 );
 	this.DEFAULT_CAMERA.lookAt( new THREE.Vector3() );
+	this.DEFAULT_MATERIAL = new THREE.MeshBasicMaterial( {
+		color: 0xff00ff
+	} );
+	this.DEFAULT_MATERIAL.uuid = 'AFE5AF61-A88E-493B-90D1-6AB205395C0B';
```

Comment:
I have added option to delete material from the material browser and fallback to default editor material.
@mrdoob can you please review the change for the default material and let me know your thoughts on it.